pom.xml

```diff
@@ -517,7 +525,12 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-failsafe-plugin</artifactId>
-					<version>2.12</version>
+					<version>3.0.0-M4</version>
```

Comment:
Anything we might need to consider before bumping these versions? Can they be transitive in some way that might lead to other issues?

Should be consider the latest version for 2.x instead of new major version?

Reply:
I don't think these versions are transitive (see comment above). About using `2.x` version: I think that if the build works with `3.x`, then this is better since we are more prepared for future changes in jdk.